app/views/spree/users/show.html.haml

```diff
@@ -7,6 +7,34 @@
       .account-summary{"data-hook" => "account_summary"}
         = @user.email
         (#{link_to t(:edit), spree.edit_account_path})
+      %hr
+      %h3= t(:my_addresses)
```

Comment:
When I did the internationalisation first, it was easy to write `t(:my_addresses)`. But in most views it's better to use translations in a local scope: `t('.my_addresses')`. The locale then needs an entry like:
```yml
en:
  spree:
    users:
      show:
        my_address: 'My Addresses'
```

This way you avoid any conflicts between views and the locale file is more structured. What do you think?